### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by Andrew-Dev-Coder** [Andrew-Dev-Coder/silly._.sketches_ect#4]
Currently alt text is "an error has occured" this should be changed to be a description of the image.
